src/plugins/intel_cpu/src/output_mem_mgr.cpp

```diff
@@ -0,0 +1,71 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "output_mem_mgr.h"
+
+using namespace ov::intel_cpu;
+
+void* OutputMemoryMngr::getRawPtr() const noexcept {
+    if (m_allocator) {
+        return _data;
+    } else {
+        return _pMemMngr->getRawPtr();
+    }
+}
+
+void OutputMemoryMngr::setExtBuff(void* ptr, size_t size) {
+    if (m_allocator) {
+        return;
```

Comment:
Please avoid silent action skipping. Either replace the memory pointer in the allocator, or throw.